k8s/volumes.yaml

```diff
@@ -0,0 +1,120 @@
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: jasminegraph-metadb
+  labels:
+    type: local
+spec:
+  storageClassName: manual
+  capacity:
+    storage: 200Mi
```

Comment:
Storage volume for MetaDB can be safely kept at event 100Mi. I do not think it will have huge amount of data to maintain.

Reply:
Okay